libinteract/libinteract.py

```diff
@@ -1457,210 +1460,244 @@ def parse_hbs_file(fname):
 
     acceptors = cfg.get(hbs_str, acceptors_str)
     acceptors = [i.strip() for i in acceptors.strip().split(",")]
+    acceptors = "name " + " or name ".join(acceptors)
 
     donors = cfg.get(hbs_str, donors_str)
     donors = [i.strip() for i in donors.strip().split(",")]
+    donors = "name " + " or name ".join(donors)
 
-    return dict(ACCEPTORS = acceptors, DONORS = donors)
+    hydrogens = cfg.get(hbs_str, hydrogens_str)
+    hydrogens = [i.strip() for i in hydrogens.strip().split(",")]
+    hydrogens = "name " + " ".join(hydrogens)
+
+    return dict(ACCEPTORS = acceptors,
+                DONORS = donors,
+                HYDROGENS = hydrogens)
 
 
 def do_hbonds(sel1,
               sel2,
               pdb,
               uni,
-              update_selection1 = True,
-              update_selection2 = True,
-              filter_first = False,
-              distance = 3.0,
-              angle = 120,
-              perco = 0.0,
-              perresidue = False,
-              do_fullmatrix = False,
-              other_hbs = None):
-    
-    # Import the hydrogen bonds analysis module
-    from MDAnalysis.analysis.hbonds import hbond_analysis
+              update_selections,
+              d_a_cutoff,
+              d_h_a_angle_cutoff,
+              perco,
+              do_fullmatrix,
+              other_hbs):
+    
+    # Import the hydrogen bonds analysis module (MDAnalysis v1.0.0 and newer)
+    from MDAnalysis.analysis.hydrogenbonds import hbond_analysis
```

Comment:
this should just be at the top of the file with the other imports

Reply:
I will move it there.